ruff.schema.json

```diff
@@ -957,6 +957,14 @@
             "type": "string"
           }
         },
+        "lines-after-imports": {
+          "description": "The number of blank lines to place after imports. -1 for automatic determination",
```

Comment:
I'm tempted to use `null` as the automatic determination value... But, I guess in that case, we wouldn't be able to tell the difference between a config using the default value and the config using the automatic value, in the event that we need to merge two configuration files (via `extend`).